### DISCUSSION TITLE
[Feature]: karma-rspack

### DISCUSSION THREAD

**Discussion by birkskyum** [web-infra-dev/rspack#7648] [Q&A]
### What problem does this feature solve?

In order to migrate from karma-webpack (or karma-esbuild) to rspack, it would be very convenient to have a karma-rspack

### What does the proposed API of configuration look like?

As close as possible to [karma-webpack](https://www.npmjs.com/package/karma-webpack)

**Comment by hardfist** [comment]
you can try make one and if you met problems we can help but we dont have manpower to maintain this

**Comment by rcesar** [comment]
> ### What problem does this feature solve?
> In order to migrate from karma-webpack (or karma-esbuild) to rspack, it would be very convenient to have a karma-rspack
> 
> ### What does the proposed API of configuration look like?
> As close as possible to [karma-webpack](https://www.npmjs.com/package/karma-webpack)

I created this solution: https://www.npmjs.com/package/karma-rspack-2

**Reply by Kingo64** [reply]
Any chance you could update it to support Rspack 2? It's no longer compatible.
